Client/Application/TestAppDelegate.swift

```diff
@@ -8,6 +8,8 @@ import XCGLogger
 
 private let log = Logger.browserLogger
 
+// swiftlint:disable force_try force_cast
```

Comment:
Usually if I disable a rule explicitly like this there's one of two reasons: 
1. It's a test and won't run in production code
2. It's something that can't be helped due to some Obj-C interop or some legacy stuff (i.e A lot of my disables are on casting a `UITableViewCell`/`UICollectionViewCell` to their actual dequeued type.  We could remedy this will `guard let`'s but its a bit pointless... A better solution would be to migrate to something a bit smarter like https://github.com/mergesort/GenericCells but it would require a decent amount of work to refactor